packages/wifi-signal-strength/default.nix

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  stdenvNoCC,
+  pkgs,
+  lib,
+  ...
+}: let
+  wifiSignalStrength =
+    pkgs.writeShellScript
+    "wifi-signal-strength"
+    ''
+      export DBUS_SESSION_BUS_ADDRESS=unix:path=/tmp/ssh_session_dbus.sock
+      export DBUS_SYSTEM_BUS_ADDRESS=unix:path=/tmp/ssh_system_dbus.sock
+      ${pkgs.openssh}/bin/ssh -M -S /tmp/nmcli_socket \
+          -f -N -q ghaf@192.168.100.1 \
+          -i /run/waypipe-ssh/id_ed25519 \
+          -o StrictHostKeyChecking=no \
+          -o StreamLocalBindUnlink=yes \
+          -o ExitOnForwardFailure=yes \
+          -L /tmp/ssh_session_dbus.sock:/run/user/1000/bus \
+          -L /tmp/ssh_system_dbus.sock:/run/dbus/system_bus_socket
+      signal0=󰤟
+      signal1=󰤢
+      signal2=󰤥
+      signal3=󰤨
+      no_signal=󰤭
+      # Get IP address of netvm
+      address=$(${pkgs.networkmanager}/bin/nmcli device show wlp0s5f0 | ${pkgs.gawk}/bin/awk '{ if ($1=="IP4.ADDRESS[1]:") {print $2}}')
```

Comment:
Same here with the NIC device name `wlp0s5f0` - it's reference hardware specific and should be passed as argument to `callPackage` via config from the target HW definition.

Reply:
Updated to get "wlp" device as in wifi-connector script instead to pass as argument. If it is more stable to passed as an argument, I can also apply it.

Reply:
The problem with this implementation https://github.com/tiiuae/ghaf/pull/386/files#diff-0dd9147b12941844c66d8519a708fb64e934748abb257ff28c09de8bec9dc667R31 is that it's not usable with other reference devices.
I think it would be better to have something like:
```
targets/lenovo-x1-carbon.nix:
  wifiDevice = "wlp0s5f0";
```
and then pass that as argument to `packages/wifi-signal-strength/default.nix`

Reply:
The name is moved to hardware definition.

Reply:
You probably forgot to `--amend` the changed file to the commit as it is not visible in the "Files changed" nor in the latest [compare](https://github.com/tiiuae/ghaf/compare/17d5f22f48901564353908475828e3951151fdb5..ef25d002c11a67d2cb828ca79268e8fa39d44a13).

Reply:
Oh, yes. My bad, sorry. Fixed now. :)